### ISSUE TITLE
Context menu option to copy bookmark text in the bookmark bar and manager

### ISSUE THREAD

**Issue by dentistformyeye** [brave/brave-browser#19956]
## Description 

I want to copy bookmark text without copying the URL. In the context menu from right-clicking a bookmark in the bookmark bar, please add a "Copy text" option and change "Copy" to "Copy URL". When right-clicking a bookmark in the bookmark manager the context menu has both "Copy" and "Copy URL", but both of them simply copy the URL. Please change "Copy" to "Copy text" and change the behavior accordingly.

## Steps to Reproduce

   1. Right-click a bookmark in the menu bar. (There is no "Copy text" option yet.) Close the context menu.
   1. Go to the bookmark manager brave://bookmarks
   2. Right-click a bookmark
   3. Click "Copy"
   4. Paste in the address bar

## Actual result:

There is no option to copy text in the bookmark bar context menu. The "Copy" option and "Copy URL" option in the bookmark manager context menu both do the same thing. Step 5 pastes the bookmark URL.

## Expected result:
In the bookmark bar context menu there is a "Copy text" option and the "Copy" option is renamed to "Copy URL". In the bookmark manager context menu "Copy" is instead "Copy text".
(Selecting a bookmark in the bookmark manager and using ctrl-c should still copy the URL.)

## Reproduces how often: 

Easily reproduced

## Brave version (brave://version info)

Brave | 1.32.106 Chromium: 96.0.4664.45 (Official Build) (x86_64)
-- | --
Revision | 76e4c1bb2ab4671b8beba3444e61c0f17584b2fc-refs/branch-heads/4664@{#947}
OS | macOS Version 12.0.1 (Build 21A559)

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Haven't tried
- Can you reproduce this issue with the nightly channel? Haven't tried

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? N/A
- Does the issue resolve itself when disabling Brave Rewards? N/A
- Is the issue reproducible on the latest version of Chrome? Haven't tried

**Comment by rebron** [comment]
@dentistformyeye Selecting Edit and copying the text from there is not enough? This doesn't seem like a common enough case to expand the context menu vs using the Edit menu which allows you to actually even see the bookmark text that you're copying and allowing you to change that text selection.

**Comment by dentistformyeye** [comment]
That's true. A "Copy text" option isn't strictly necessary for the context menu from the bookmark bar.
However, the "Copy" option in the context menu from the bookmark manager should still be modified because it's unexpected for two buttons to do the exact same thing.
